tests/php/Forms/GridField/GridFieldDetailForm_ItemRequestTest.yml

```diff
@@ -0,0 +1,8 @@
+SilverStripe\Forms\Tests\GridField\GridFieldTest\Team:
+    team1:
```

Comment:
Indentation in this file is wrong, should be 2 spaces

Also add a blank line at end of the file

Reply:
Updated